content/command-line/concepts/bash/terms/input-output-redirection/input-output-redirection.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Bash: Input-Output Redirection'
+Description: 'Redirection allows the modification of the destination of standard input and standard output in a shell.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Computer Science'
+Tags:
+  - 'Bash/Shell'
+  - 'Command Line'
+  - 'Linux'
+  - 'Redirection'
```

Comment:
Tags should be picked from the list in [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md). `'Bash/Shell'` and `'Redirection'` are not valid tags.

```suggestion
  - 'Command Line'
  - 'Linux'
```